packages/resolver/lib/sources/fs.ts

```diff
@@ -65,18 +65,21 @@ export class FS implements ResolverSource {
     ];
 
     let body, filePath;
-    possiblePaths.forEach(possiblePath => {
+    for (const possiblePath of possiblePaths) {
       try {
         const resolvedSource = fs.readFileSync(possiblePath, {
           encoding: "utf8"
         });
         body = resolvedSource;
         filePath = possiblePath;
+        
+        return { body, filePath };
```

Comment:
@gorgos , in terms of making this fix as clean as possible and getting it out the door, it looks like this line is all that needs to be added.

Reply:
Well, and the change to `for .. of` loop.